apps/storybook-react/vitest.config.ts

```diff
@@ -0,0 +1,41 @@
+import { storybookTest } from '@storybook/addon-vitest/vitest-plugin';
```

Comment:
Setting up vitest according to storybook v9 docs https://storybook.js.org/docs/writing-tests/integrations/vitest-addon I realize it seems like a lot of the same files but this is correct